package.json

```diff
@@ -68,7 +68,6 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@cumulus/aws-client": "^1.24.0",
```

Comment:
remove the unnecessary old package which could cause the compile error and audit error,
add missing packages for packages/async-operations/package.json, lambdas/postgres-migration-count-tool/package.json